go.mod

```diff
@@ -1,36 +1,36 @@
 module github.com/openshift/node-observability-operator
 
-go 1.17
+go 1.18
```

Comment:
Looking at the logs of the CI I see that `golang-1.18` image is still used for the builds,seems like `build_root` field from the CI config was exactly for this. However the operator image is built from `Dockerfile.rhel8` image ([link](https://github.com/openshift/release/blob/918f7fefade600a73f960056167d772f4ca35eb8/ci-operator/config/openshift/node-observability-operator/openshift-node-observability-operator-main.yaml#L23-L30)) which uses golang 1.19.
We should avoid this discrepancy. Since `build_root` doesn't have 1.19 yet we can try to instruct CI to look for the build image in the repository. For instance, similar to the way it's done in extdns operator: [repo](https://github.com/openshift/external-dns-operator/blob/main/.ci-operator.yaml), [CI config](https://github.com/openshift/release/blob/918f7fefade600a73f960056167d772f4ca35eb8/ci-operator/config/openshift/external-dns-operator/openshift-external-dns-operator-main.yaml#L18-L19).

Reply:
Related PRs: 
1. https://github.com/openshift/node-observability-operator/pull/108
2. https://github.com/openshift/release/pull/33281